### PR TITLE
[TIMOB-23851] (6_0_X) Fixed iOS module lipo call and arch verification.

### PR DESCRIPTION
**Pull request by cb1kenobi**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-23851

### CODE REVIEW

**Comment by hansemannn** [comment]
```
[DEBUG] Running: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo /Users/hans/Documents/appcelerator_modules/ti.facebook/ios/build/Release-iphoneos/libfacebook.a /Users/hans/Documents/appcelerator_modules/ti.facebook/ios/build/Release-iphonesimulator/libfacebook.a -create -output /Users/hans/Documents/appcelerator_modules/ti.facebook/ios/build/libfacebook.a
Caught exception: TypeError: Cannot read property 'error' of undefined
```

**Comment by cb1kenobi** [comment]
@hansemannn Whoops. Fixed.

File: iphone/cli/commands/_buildModule.js [hunk 1]
```diff
@@ -436,33 +441,36 @@ iOSModuleBuilder.prototype.buildModule = function buildModule(next) {
...
-		];
-
-	this.dirWalker(this.universalBinaryDir, function (file) {
-		if (path.extname(file) === '.a'
-			&& file.indexOf( this.moduleName + '.build') === -1
-			&& file.indexOf('Release-') > -1
-		) {
-			binaryFiles.push(file);
+	var args = [
+		path.join(this.projectDir, 'build', 'Release-iphoneos', 'lib' + this.moduleId + '.a'),
```
Comment (feons) [inline iphone/cli/commands/_buildModule.js]: Shouldn't this be `this.moduleName`?

**Comment by feons** [comment]
LGTM!

**Pull request merged**